### DISCUSSION TITLE
Change search parameter

### DISCUSSION THREAD

**Discussion by Tychodewaard** [vanjarosoftware/Vanjaro.Platform#1532] [Q&A] [answered]
By default the search generates a url like /Search-Results?Search=keyword.
Where/how do I change what to make it /Search-Results?sb-search=keyword ?

**Comment by meetmandeep** [comment, marked as answer]
There is no option to override or change that parameter at this time.

**Comment by Tychodewaard** [comment]
I know but can it be done custom? Because it concerns a theme that is already custom (you company made it 😀)

Outlook voor iOS<https://aka.ms/o0ukef> downloaden
________________________________
Van: Mandeep Singh ***@***.***>
Verzonden: Monday, July 19, 2021 2:59:47 PM
Aan: vanjarosoftware/Vanjaro.Platform ***@***.***>
CC: Tycho de Waard ***@***.***>; Author ***@***.***>
Onderwerp: Re: [vanjarosoftware/Vanjaro.Platform] Change search parameter (#1532)


There is no option to override or change that parameter at this time.

**Reply by meetmandeep** [reply]
That's a core Vanjaro change and cannot be done with a custom theme. Add it to issues as a enhancement or feature request for future consideration.

**Reply by Tychodewaard** [reply]
ok. For other users running in the same thing: This usecase was to use SearchBoost in a Vanjaro setup. The way to implement SearchBoost in Vanjaro:
1. Create a results page in Vanjaro
2. add Search Boost results module to search results page
3. Change the SearchBoost parameter from sb-search to Search

Note: do not use the SearchBoost search input as it is not compatible with Vanjaro (or one of its components).
